.gitignore

```diff
@@ -14,3 +14,7 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+.DS_Store
```

Comment:
This isn't application specific so should probably be left for a global configuration such as [https://github.com/github/gitignore/blob/master/Global/macOS.gitignore]